source/chapters/how_to_map_controllers.rst

```diff
@@ -0,0 +1,23 @@
+.. include:: /shortcuts.rstext
+
+.. _advanced-controller:
+
+How to add support for new Controllers
+***************************************
+
+With several dozens of DJ controllers supported out-of-the-box, Mixxx gives you
+comprehensive hardware control for your DJ mixes, see :ref:`control-midi`.
+
+Support for additional devices can be added to Mixxx by creating a new mapping
+file. This file tells Mixxx how to translate, or map, :term:`MIDI`/:term:`HID`
+messages from a controller into commands that Mixxx understands.
+
+You can download and share custom controller mappings in the
+`Mixxx User customizations forums <https://mixxx.org/forums/viewforum.php?f=6>`_.
```

Comment:
Instead of redirecting, let's use the new link right away
https://mixxx.discourse.group/c/controller-mappings/10